src/datasource.ts

```diff
@@ -35,17 +35,17 @@ export default class VividCortexDatasource {
         title: 'Credentials error',
       };
 
-    return this.doRequest('metrics', 'GET', { limit: 1 }).then(
-      response => {
+    return this.doRequest('metrics', 'GET', { limit: 1 })
+      .then(response => {
         if (response.status === 200) {
           return success;
         }
         return error;
-      },
-      () => {
+      })
+      .catch(response => {
+        console.error(response);
```

Comment:
@matyax Can you please confirm that this is not a simple debug message that you forgot to remove?

Reply:
I confirm that. It will help Customer Support and us to give better answers for customers whenever the datasource stops working.

Reply:
thanks for that :)